### ISSUE TITLE
Bug on EJS files with grunt build 

### ISSUE THREAD

**Issue by jguyomard** [gmetais/YellowLabTools#189]
Hi,

Thank you for your job!

Since #186, i've a bug with the minified version of YLT. With `grunt build`, the `main.html` file is minified like this : 

```html
<base href="&lt;%= baseUrl %&gt;">
[...]
<a href="&lt;%= baseUrl %&gt;about">More about Yellow Lab Tools</a>
```

**Comment by gmetais** [comment]
Hi @jguyomard,

Sorry about the issue and thank you for reporting it.

I've not found a way to fix this in grunt-htmlmin yet. A quick fix for the moment is to ignore the latest commit:
```
git checkout c82c1c192f9fb49f18b4c8206b6bfb95fd8f1d2b
```

**Comment by gmetais** [comment]
Fixed by #192.
